docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -5,7 +5,6 @@ Cache filter
 
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.cache.v3.CacheConfig>`
 * :ref:`v3 SimpleHTTPCache API reference <envoy_v3_api_msg_extensions.cache.simple_http_cache.v3.SimpleHttpCacheConfig>`
-* This filter should be configured with the name ``envoy.filters.http.cache``.
```

Comment:
maybe im not understanding something here

```console
$ envoydev/envoy/examples/cache$ git diff
```

```diff
diff --git a/examples/cache/front-envoy.yaml b/examples/cache/front-envoy.yaml
index b953d1e781..caa0ee8b9f 100644
--- a/examples/cache/front-envoy.yaml
+++ b/examples/cache/front-envoy.yaml
@@ -27,8 +27,7 @@ static_resources:
                 route:
                   cluster: service2
           http_filters:
-          - name: "envoy.filters.http.cache"
-            typed_config:
+          - typed_config:
               "@type": "type.googleapis.com/envoy.extensions.filters.http.cache.v3.CacheConfig"
               typed_config:
                 "@type": "type.googleapis.com/envoy.extensions.cache.simple_http_cache.v3.SimpleHttpCacheConfig"
```

```console
$ ./verify.sh
...
front-envoy_1    | [2022-07-13 12:50:15.726][9][critical][main] [source/server/server.cc:117] error initializing configuration '/etc/front-envoy.yaml': Proto constraint validation failed (HttpConnectionManagerValidationError.HttpFilters[0]: embedded message failed validation | caused by HttpFilterValidationError.Name: value length must be at least 1 characters): stat_prefix: "ingress_http"
...
```

ok i see

```diff
diff --git a/examples/cache/front-envoy.yaml b/examples/cache/front-envoy.yaml
index b953d1e781..36bcbcb27b 100644
--- a/examples/cache/front-envoy.yaml
+++ b/examples/cache/front-envoy.yaml
@@ -27,7 +27,7 @@ static_resources:
                 route:
                   cluster: service2
           http_filters:
-          - name: "envoy.filters.http.cache"
+          - name: "my.own.special.filter.name"
             typed_config:
               "@type": "type.googleapis.com/envoy.extensions.filters.http.cache.v3.CacheConfig"

```

...works, hmmm